sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -229,6 +229,51 @@ abstract class JdbcDialect extends Serializable with Logging{
     }
   }
 
+  /**
+   * Create schema with an optional comment. Empty string means no comment.
+   */
+  def createSchema(
+      conn: Connection, options: JDBCOptions, schema: String, comment: String): Unit = {
+    val statement = conn.createStatement
```

Comment:
shall we pass `statement` as parameter to `createSchema `? Then Spark can set the timeout and close the statement at the end, leave less job to the JDBC dialect implementations.

Reply:
OK